README.md

```diff
@@ -117,18 +117,18 @@ All instructions going forward will assume that the environment variables have b
 
 ### Prepare Database
 
-1. Prepare the database: `docker compose run web /bin/bash -c "source venv/bin/activate && yarn init-db --populate_indicators"`
+1. Prepare the database: `docker compose run --rm web /bin/bash -c "source venv/bin/activate && yarn init-db --populate_indicators"`
 
 ### Run Web Server
 1. Start your development environment: `docker compose up`
-2. In a separate terminal, create user account `docker compose exec web /bin/bash -c "source venv/bin/activate && ./scripts/create_user.py -u me@mydomain.com -p password --first_name=admin --last_name=istrator -a"`
+2. In a separate terminal, create user account `docker compose exec web /bin/bash -c "source venv/bin/activate && ./scripts/create_user.py -u me@mydomain.com --password password --first_name=admin --last_name=istrator --site_admin"`
```

Comment:
`-u` -> `--username` for consistency

also can we either do the `--password password` or `--password=password` syntax, but not both?

Reply:
Thanks for catching that!